kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -70,8 +70,168 @@
+        >
+          <div
+            v-for="item in placeholderList"
+            :key="item.id"
+          >
+            <AccordionItem
+              :id="item.id"
+              :key="item.id"
+              :items="placeholderList"
+              :title="item.title"
+              :expanded="isItemExpanded(item.id)"
+              @click="toggleItemID(item.id)"
+            >
+              <template
+                #heading="{ title }"
+                :accordionToggle="onAccordionToggle(item.id)"
+              >
+                <KGrid class="question-row">
+                  <KGridItem :layout12="{ span: 6 }">
+                    <div class="left-column-alignment-style">
+                      <DragHandle>
+                        <KIcon
+                          class="drag-icon icon-size"
+                          icon="dragVertical"
+                        />
+                      </DragHandle>
+
+                      <div>
+                        <p>
+                          <KCheckbox />
+                        </p>
+                      </div>
+
+                      <div>
+                        <p>
+                          {{ title }}
+                        </p>
+                      </div>
+                    </div>
+                  </KGridItem>
+
+                  <KGridItem :layout12="{ span: 6 }">
+                    <div class="right-alignment-style">
+                      <KIconButton
+                        v-if="isItemExpanded(item.id)"
+                        class="icon-size"
+                        icon="chevronUp"
+                        @click="toggleItemState(item.id)"
+                      />
+
+                      <KIconButton
+                        v-else
+                        class="icon-size"
+                        icon="chevronRight"
+                        @click="toggleItemState(item.id)"
+                      />
+                    </div>
+                  </KGridItem>
+                </KGrid>
+              </template>
+
+              <template
+                v-if="isItemExpanded(item.id)"
+                #content
+              >
+                <div class="accordion-detail-container">
+                  <KGrid>
+                    <KGridItem :layout12="{ span: 8 }">
+                      <p>{{ $tr('questionPhrase') }}</p>
+                      <p>{{ $tr('questionSubtitle') }}</p>
+                    </KGridItem>
+
+                    <KGridItem :layout12="{ span: 4 }">
+                      <KIcon class="float-item-left-style" icon="edit" />
+                    </KGridItem>
+                  </KGrid>
+
+                  <div class="choose-question question">
+                    <p class="space-content">
+                      {{ $tr('chooseQuestionLabel') }}
+                    </p>
+                  </div>
+
+
+                  <hr class="bottom-border">
+                  <KButton
+                    style="width:100%;margin-bottom:10px"
```

Comment:
My note elsewhere re: typically using 8px increments should also be applied to inline styles in the templates